public/css/styles.css

```diff
@@ -1608,6 +1608,47 @@ svg > g > g:last-child { pointer-events: none }
   border: 1px solid rgb(44, 151, 250);
 }
 
+.tabcontentscores th {
+  text-align: center;
```

Comment:
I'd like to keep table headers left aligned and normal font-weight for now to match the rest and do such changes site-wide eventually with the new stack.
Can stay as is as well though, as you wish.